docs/en-US/component/text.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Text
+lang: en-US
+---
+
+# Text
+
+Used for text.
+
+## Basic
+
+:::demo Use the `type` attribute to define Text's type.
+
+text/basic
+
+:::
+
+## Sizes
+
+:::demo Use attribute `size` to set additional sizes with `large`, `default` or `small`.
+
+text/sizes
+
+:::
+
+## Ellipsis
+
+:::demo Pass the `truncated` prop to render an ellipsis when the text exceeds the width of the viewport or max-width set.
+
+text/truncated
+
+:::
+
+## Override
+
+:::demo Use attribute `tag` to override element
+
+text/override
+
+:::
+
+## Mixed
+
+:::demo Text mixed component
+
+text/mixed
+
+:::
+
+## API
+
+### Attributes
+
+| Name      | Description        | Type                                                               | Default |
+| --------- | ------------------ | ------------------------------------------------------------------ | ------- |
+| type      | text type          | ^[enum]`'primary' \| 'success' \| 'warning' \| 'danger' \| 'info'` | —       |
+| size      | text size          | ^[enum]`'large' \| 'default' \| 'small'`                           | default |
+| truncated | render ellipsis    | ^[boolean]                                                         | false   |
+| tag       | custom element tag | ^[string]                                                          | span    |
```

Comment:
Maybe we should call it `as`, since most UIs do.

Reply:
refer [tag attributes of Row and Col](https://element-plus.org/en-US/component/layout.html#row-attributes)

Reply:
At the beginning Reference [chakra-ui](https://vue.chakra-ui.com/text) uses `as` prop name, but el-row/col use `tag` prop name, so change prop name to `tag`. 
`tag` el-row/col used, `as` is most UIs do, difficult choice.

😵 Wait for @jw-foss  and @tolking  discuss the results, I will change it as soon as possible.

Reply:
@gimjin let's keep consistent with `el-col` and `el-row`. Approved! /cc @tolking